README.md

```diff
@@ -109,6 +109,13 @@ compiler version is to add the following fields to your VSCode user settings:
 
 Where of course `v0.8.3+commit.8d00100c` can be replaced with any other version.
 
+## Admin Privileges
+
+Users of the Tranche contracts should note that there are some limited admin permissions. These will be controlled by a decentralized governance mechanism and include:
+
+- Ability to manually "mature" bonds, disabling deposits and distributing collateral to the tranche token holders
```

Comment:
Manual maturation is determined by the bond creator, not the owner of BondFactory

Reply:
oh. true. fixed